app/assets/javascripts/pages/static_pages/search.js

```diff
@@ -1,45 +1,35 @@
 (function (App) {
-    App.Page.SearchPage = Backbone.View.extend({
+  App.View.Search = Backbone.View.extend({
+    initialize: function () {
+      this._setVars();
+      this._setEventListeners();
+    },
+
+    _setVars: function() {
+      this.$searchContainer = $('#search-container');
+      this.$searchButton = $('#search-button');
+      this.$closeButton = $('#close-button');
+      this.$searchInput = $('#search-input');
+    },
+
+    _setEventListeners: function () {
+      this.$searchInput = $('#search-input');
+      this.$searchButton.on('click', this._toggleSearch.bind(this));
+      this.$closeButton.on('click', this._closeSearch.bind(this));
+    },
+
+    _toggleSearch: function () {
+      this.$searchContainer.toggleClass('-open');
+      if (this.$searchContainer.hasClass('-open')) {
+        $('.c-modal-search').bind('transitionend webkitTransitionEnd oTransitionEnd MSTransitionEnd', function(e){
```

Comment:
why not `on` instead `bind`?

Reply:
didn't know .bind() pointed to .on() until now . changed!!